### PR TITLE
DF 797 PoBox address logic

### PR DESCRIPTION
**Pull request by emina-str**
- remove mapping of city and postal code (moved to backend)

Backend: https://github.com/alv-ch/job-ad-service/pull/321

### CODE REVIEW

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts [hunk 1]
```diff
@@ -389,6 +377,7 @@ function mapToLocation(locationFormValue: LocationFormValue): CreateLocation {
 }
 
 function mapToCompany(companyFormValue: CompanyFormValue, surrogate: boolean): Company {
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts]: In the backend it is WebformCreateCompanyDto, what doesn't match with Company

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts [hunk 2]
```diff
@@ -462,20 +450,12 @@ function mapToApplyChannelPostAddress(postAddressFormValue: PostAddressFormValue
...
-  if (postOfficeBoxNumber) {
-    return {
-      name: postAddressFormValue.name,
-      countryIsoCode: postAddressFormValue.countryIsoCode,
-      postOfficeBoxNumber,
-      ...mapToPostOfficeBoxPostalCodeAndPostOfficeBoxCity(postOfficeBoxNumber, postAddressFormValue.zipAndCity)
-    };
-  }
-
   return {
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts]: In the backend it is WebformCreateAddresDto, what doesn't match with PostAddress

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts (hunk 1 above)
Comment (emina-str) [reply alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts]: fixed

File: alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts (hunk 2 above)
Comment (emina-str) [reply alv-portal-ui/src/app/job-advertisement/job-publication/job-publication-form/job-publication-form.mapper.ts]: fixed

**Pull request merged**